ethereum/contracts/Getters.sol

```diff
@@ -53,4 +53,20 @@ contract Getters is State {
     function nextSequence(address emitter) public view returns (uint64) {
         return _state.sequences[emitter];
     }
+
+    function getGuardianSetHash(uint32 index) public view returns (bytes32) {
+        return _state.guardianSetHashes[index];
+    }
+
+    function getEncodedGuardianSet(uint32 index) public view returns (bytes memory encodedGuardianSet) {
+        Structs.GuardianSet memory guardianSet = getGuardianSet(index);
+
+        // Encode the guardian set.
+        uint256 guardianCount = guardianSet.keys.length;
+        for (uint256 i = 0; i < guardianCount;) {
+            encodedGuardianSet = abi.encodePacked(encodedGuardianSet, guardianSet.keys[i]);
```

Comment:
This can be optimized by introducing inline assembly, however, this method is designed to called off-chain.

Reply:
Perhaps it's worth adding that as a comment in the code?

Reply:
Yes, definitely. I still need to add documentation to this PR.